src/ui/MD/components.tsx

```diff
@@ -70,7 +70,7 @@ const fixAlign = (align: React.CSSProperties["textAlign"]): TableCellProps["alig
   if (align === "end") return "inherit";
   if (align === "start") return "inherit";
   if (align === "match-parent") return "inherit";
-  return align;
+  return align as TableCellProps["align"];
```

Comment:
Without this we get this build failure, perhaps from changes in some of the dependency versions since this workaround was made

https://github.com/bitburner-official/bitburner-src/actions/runs/13950987128/job/39050155791?pr=2037